lib/components/Modal.js

```diff
@@ -7,12 +7,15 @@ var elementClass = require('element-class');
 var renderSubtreeIntoContainer = require("react-dom").unstable_renderSubtreeIntoContainer;
 
 var SafeHTMLElement = ExecutionEnvironment.canUseDOM ? window.HTMLElement : {};
+var AppElement = document.body;
```

Comment:
@azaharakis

I don't have experience with isomorphic apps, but I guess it's because document.body doesn't exist at compile time. Would it work if I do it like this instead?

````javascript
    var AppElement;

    componentDidMount: function() {
        ...
        (AppElement || document.body).appendChild(this.node);
        ...
    },
````